src/components/global/search/search.hbs

```diff
@@ -1,5 +1,8 @@
-<div class="search">
-	<h2 class="visuallyhidden">Suche</h2>
+<div class="search js-search">
+	<h2 class="visuallyhidden">Search</h2>
+	<button class="search--toggle">
+    <span class="visuallyhidden">Search</span>
+  </button>
 	<input class="search--input" type="text">
```

Comment:
This input doesn't have an ID, so it can't have a `<label>`. But every input needs a label.